examples/gluon_alexnet/README.md

```diff
@@ -0,0 +1,219 @@
+## Loading and serving Gluon models on MXNet Model Server (MMS)
+
+We are glad to announce that MXNet Model Server now supports loading and serving Imperative and Hybrid Gluon models. 
+This is a short tutorial on how to write a custom Gluon model and serve it on MMS. If you are using Gluon for the first
+time, you could refer this [60 min crash-course](http://gluon-crash-course.mxnet.io/ndarray.html) on gluon to get a 
+better understanding.
+
+In this tutorial, we will cover the following:
+1. [How to serve Gluon models](#how-to-serve-gluon-models)
+2. [Load and serve pretrained models](#load-and-serve-a-pretrained-gluon-model)
+3. [Load and serve pure Gluon imperative models](#load-and-serve-pure-gluon-imperative-models)
+4. [Load and serve a hybrid'ized gluon model](#load-and-serve-a-hybrid'ized-gluon-model)
+5. [How to test your gluon models](#how-to-test-your-gluon-models)
+6. [How to create a model archive](#how-to-create-a-model-archive)
+
+**NOTE: If you do not use `Symbols` parameter in `MANIFEST.json`, MMS infers this as a imperative gluon model
+(or network defined by run). In other words DO-NOT-USE `Symbols` parameter in `MANIFEST.json` when defining `Gluon` 
+models.** 
+
+## How to serve Gluon models
+There are three ways of defining our Gluon nerual network/model.
+1. Load and serve a pretrained model.
+2. Load and serve custom imperative model, which implies that we don't generate/use symbols.
```

Comment:
Why call it "custom"? This is just a "Gluon imperative model". I think this is a simple way to put it, no need to confuse with "generate/use symbols".
Think about the average user who may not be aware of all of this.